src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
```

Comment:
move lottery creation out of the loop
```suggestion
            Lottery lottery = new Lottery();
```